### ISSUE TITLE
Implementation of a baseline 

### ISSUE THREAD

**Issue by sdat2** [ai4er-cdt/gtc-biodiversity#9]
EDIT:
Remaining things to be done:

- [x] Get ESA CCI data as test data
- [x] Define region of interest
- [x] Clip raster to region of interest
- [x] Separate zones
- [x] Reproject to suitable CRS
- [x] Perform pylandstats analysis on timeseries from 1992 to 2015
- [x] Sanity checks
- [ ] Transfer final pipeline code from jupyter notebooks to python script
- [ ] Finalize plotting procedures and add to python script

**Comment by Croydon-Brixton** [comment]
Purpose:
Create a baseline of the currently existing fragmentation algorithms (probably starting from land cover data)

Baseline:
- Percolation theory (Nature paper)
- Papers by Fahrig (Fahrig 2003, Jaeger 2000)

**Comment by Croydon-Brixton** [comment]
@arduinfindeis  @KMacfarlaneGreen 
I opened a branch for us to work on this: `feature/fragmentation-baseline`

**Comment by Croydon-Brixton** [comment]
Update:
I started to add a few convenience scripts for working with rasterio and plotting land-cover data, as I imagine we will use this a lot.

For an explanation of the first steps with Rasterio, see [this notebook](https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/blob/feature%2Ffragmentation-baseline/notebooks%2Fexploratory%2Fsvm-1-landcover-ceda-exploration.ipynb/file).

@KMacfarlaneGreen has already done a first evaluation of some fragmentation characteristics via pylandstat around the region of London in [this notebook here](https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/blob/feature%2Ffragmentation-baseline/notebooks%2Fexploratory%2F1.0_kmg_landscape-metrics-example.ipynb/file) - thanks Katie! 

Next step will be to compute this efficiently without having to do to-disk memory operations, so that we can eventually do it on larger areas (pylandstats is also quite slow at the moment).

**Comment by Croydon-Brixton** [comment]
Update:
We now have the geometry of the exclusion zone, as well as buffer zones (30 km & 60 km) loaded from Open Street Map.
Further, the ESA CCI Landcover data from 1992-2015, cut to our region of interest, is available in our joint group workspace directory.

This means we can now properly start the fragmentation analysis covering the years 1992-2015.

For more information, see [this notebook](https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/blob/feature/fragmentation-baseline/notebooks/exploratory/svm-2-first-fragmentation-analysis.ipynb/)

**Comment by Croydon-Brixton** [comment]
Update:
Spatiotemporal analysis in zone is performed in [notebook no. 5](https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/blob/feature/fragmentation-baseline/notebooks/exploratory/svm-5-pylandstats-fragmentation-analysis.ipynb/) and saved to disk in `netCDF` form. It can be loaded via `xarray` as shown [here](https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/blob/feature/fragmentation-baseline/notebooks/exploratory/svm-6-fragmentation-analysis-plotting.ipynb/) to plot.

The next step in this workstream is to produce some compelling visualizations of a few key metrics to guide our general analysis.

Once that is done, this workstream is complete.

**Comment by Croydon-Brixton** [comment]
Remaining things to be done:

- [ ] Transfer final pipeline code from jupyter notebooks to python script
- [ ] Finalize plotting procedures and add to python script
